lib/sift/value_parser.rb

```diff
@@ -57,5 +57,26 @@ def boolean_value
         ActiveRecord::Type::Boolean.new.type_cast_from_user(value)
       end
     end
+
+    def normalized_value(value, type)
+      @value = value
```

Comment:
I would prefer not using an instance variable here and instead just pass value into the method below
ie: `normalized_date_range(value)`

Reply:
In this case, I think we'd need to augment the `parse_as_range?` method to take a value argument as well, since we'd be calling that before the value is set.

Reply:
Ah yes. I don't see a way around setting the value of the instance variable twice without adding that optional parameter. I'm happy going either way.